tests_operations.py

```diff
@@ -50,10 +51,13 @@ def test_division_exception_on_zero():
 
 def test_exponentiation():
     """
-    GIVEN: Two numbers
-    WHEN:  passed in to the function exponentiation
-    THEN:  the resulting is the exponentiation of the two number
+    GIVEN: two numbers
+    WHEN: passed in to the function division
+    THEN: the resulting is the exponentiation of the two numbers
     """
```

Comment:
division? do you mean "exponentiation"

Reply:
Yes, didn't notice it when I copied the code from GitHub, will fix now